CHANGELOG.md

```diff
@@ -6,9 +6,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## 3.7.0
 ### Added
-- Support dependency view. [#887](https://github.com/microsoft/vscode-gradle/pull/887)
+- Support dependency view. [PR#887](https://github.com/microsoft/vscode-gradle/pull/887)
+- Support local Gradle installation. [PR#926](https://github.com/microsoft/vscode-gradle/pull/926)
 ### Changed
 - The view `Gradle Tasks` was renamed to `Gradle Projects`.
```

Comment:
was, are, is
please use the same tense?

Reply:
Sure.